engine/src/main/java/org/terasology/rendering/dag/AbstractNode.java

```diff
@@ -105,7 +107,6 @@ public String toString() {
     }
 
     public void setTaskListGenerator(RenderTaskListGenerator taskListGenerator) {
```

Comment:
No matter how hard I try another pair of eyes will always spot things I miss. And that's what code reviews are for! :smiley: